pkg/api/v1/types.go

```diff
@@ -586,8 +586,20 @@ type HTTPGetAction struct {
 	Port util.IntOrString `json:"port" description:"number or name of the port to access on the container"`
 	// Optional: Host name to connect to, defaults to the pod IP.
 	Host string `json:"host,omitempty" description:"hostname to connect to; defaults to pod IP"`
+	// Optional: Scheme to use for connecting to the host, defaults to HTTP.
+	Scheme URIScheme `json:"scheme,omitempty" description:"scheme to connect with, defaults to HTTP"`
```

Comment:
indicate valid values in the description (`must be HTTP or HTTPS, defaults to HTTP`), and rerun `hack/update-swagger-spec.sh` to update both spec files